sdks/python/apache_beam/testing/load_tests/load_test.py

```diff
@@ -183,3 +189,66 @@ def get_option_or_default(self, opt_name, default=0):
       return type(default)(option)
     except:
       raise
+
+
+class StatefulLoadGenerator(beam.PTransform):
```

Comment:
Is this the right place to put this? Should this be in `sdks/python/apache_beam/testing/synthetic_pipeline.py`?

Reply:
Hm, good point. I wasn't sure about that. I'll put this in `sdks/python/apache_beam/testing/synthetic_pipeline.py`